src/DynamoCore/Logging/AnalyticsService.cs

```diff
@@ -65,9 +67,9 @@ internal static bool IsADPOptedIn
                 {
                     return false;
                 }
-
-                return adpAnalyticsUI.IsOptedIn();
+                return adpAnalyticsUI.IsOptedIn(150,200);
```

Comment:
maybe add a private readonly OPTIN_RETRIES, OPTIN_SLEEP or maybe just a comment on the function call
`adpAnalyticsUI.IsOptedIn(/*retries*/150, /*sleep*/200);